docs/design/coreclr/botr/xplat-minidump-generation.md

```diff
@@ -79,14 +79,27 @@ The createdump utility can also be run from the command line on arbitrary .NET C
 
 `sudo createdump <pid>`
 
-    createdump [options] pid
-    -f, --name - dump path and file name. The pid can be placed in the name with %d. The default is "/tmp/coredump.%d"
-    -n, --normal - create minidump (default).
-    -h, --withheap - create minidump with heap.
+    -f, --name - dump path and file name. The %p, %e, %h %t format characters are supported. The default is '/tmp/coredump.%p'
+    -n, --normal - create minidump.
+    -h, --withheap - create minidump with heap (default).
     -t, --triage - create triage minidump.
     -u, --full - create full core dump.
     -d, --diag - enable diagnostic messages.
 
+
+**Dump name formatting**
+
+As of .NET 5.0, the following subset of the core pattern (see [core](https://man7.org/linux/man-pages/man5/core.5.html)) dump name formatting is supported:
+
+    %%  A single % character.
+    %d  PID of dumped process (for backwards createdump compatibility).
+    %p  PID of dumped process.
+    %P  PID of dumped process.
```

Comment:
Linux has distinct behavior for %p and %P , although not relevant to us now, maybe we should reserve %P in case we want to distinguish later (" %P  PID of dumped process, as seen in the initial PID namespace")

Reply:
I'll remove %P (return an error if used).